test/agency.js

```diff
@@ -0,0 +1,59 @@
+const config = require('../config');
+
+process.env.NODE_ENV = config.test_env;
+process.env.PORT = config.test_port;
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../server');
+
+const should = chai.should();
+
+chai.use(chaiHttp);
+
+describe('Agencies', () => {
+    it('should GET agencies limit 100', (done) => {
+        chai.request(server)
+            .get('/api/v1/agency')
```

Comment:
Just another one that should be a const somewhere since it is repeated many times.